src/main/java/gtPlusPlus/xmod/gregtech/api/metatileentity/implementations/base/GregtechMeta_MultiBlockBase.java

```diff
@@ -967,12 +967,28 @@ public boolean checkRecipeGeneric(
             return false;
         }
 
+        float batchMultiplier = 1.0f;
```

Comment:
The placement here will make batch-mode ignore void protection.

Reply:
That should be fine imo, people who use batch mode are very likely going to use an me output bus instead of normal buses

Reply:
If you want to continue that route it should atleast be explained to the user (eg adding it to the chat message when enabling batch mode).

Reply:
ok, will add that